openmetadata-ui/src/main/resources/ui/playwright/constant/alert.ts

```diff
@@ -0,0 +1,21 @@
+/*
+ *  Copyright 2024 Collate.
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+
+import { uuid } from '../utils/common';
+
+export const ALERT_DESCRIPTION = 'This is alert description';
+export const ALERT_UPDATED_DESCRIPTION = 'New alert description';
+export const TEST_SUITE_NAME = `0-pw-test-suite-${uuid()}`;
+export const TEST_CASE_NAME = `0-pw-test-case-${uuid()}`;
```

Comment:
Any reason for having this 0 before the name?

Reply:
Not that I can remember, it was a similar name in the cypress tests, So I just kept it as it is.